Call out some of the privacy concerns related to RFC 4122 version 1 algorithm

ctavan opened the pull request.

bcoe commented on README.md:
```diff
@@ -150,6 +150,20 @@ for a duplicate `v1` UUID to appear when generating UUIDs at a rate of 1M/second
 unlikely, [just like with `v4` UUIDs](#how-unique-are-v4-uuids) there is no practical guarantee
 that `v1` UUIDs are unique.
 
+### So v1 UUIDs potentially leak my MAC address and their creation timestamp?
```
I might make this heading something along the lines of:

```
### Are there privacy concerns related to v1 UUIDS?
```

bcoe commented on README.md:
```diff
@@ -150,6 +150,20 @@ for a duplicate `v1` UUID to appear when generating UUIDs at a rate of 1M/second
+`v1` UUIDs would indeed leak the hardware MAC address of the machine where they are being created.
+[As discussed above](#but-arent-v1-uuids-better-because-they-are-guaranteed-to-be-unique) this
+would most likely not be the case in modern implementations where hardware MAC addresses are either
+unavailable (browser) or not unique (containers) but there are
```
I would make this two sentences:

") this
would most likely not be the case in modern implementations where hardware MAC addresses are either
unavailable (browser) or not unique (containers)". However, there are rumors that the presence  of the MAC...

ctavan commented:
@bcoe thanks for your suggestions.

I modified the commit accordingly and added one additional note that as an example apparently even the most recent version of MySQL still uses the MAC address on some operating systems which I hope emphasizes the fact that this is not a purely hypothetical issue.
